data/public_data/loader.py

```diff
@@ -74,14 +74,16 @@ def load_cinema_schedule(self, cinema_schedule):
                 additional_info = cinema['type']
                 schedule_list = cinema['schedule']
                 for timing in schedule_list:
-                    try:
-                        self.cursor.execute("INSERT INTO showings (cinema_id, movie_id, type, schedule) "
-                                            "VALUES (%s, %s, %s, %s)", (cinema_id, movie_id, additional_info, timing))
-                    except psycopg2.IntegrityError:
-                        continue
-                    except psycopg2.InternalError:
-                        continue
-        self.conn.commit()
+                    self.cursor.execute("INSERT INTO showings (cinema_id, movie_id, type, schedule) "
+                                        "VALUES (%s, %s, %s, %s) "
+                                        "ON CONFLICT (cinema_id, movie_id, type, schedule) "
+                                        "DO UPDATE SET (cinema_id, movie_id, type, schedule) = (%s, %s, %s, %s) "
+                                        "WHERE showings.cinema_id=%s AND showings.movie_id=%s "
+                                        "AND showings.type=%s AND showings.schedule=%s",
+                                        (cinema_id, movie_id, additional_info, timing,
+                                         cinema_id, movie_id, additional_info, timing,
+                                         cinema_id, movie_id, additional_info, timing))
```

Comment:
> cinema_id, movie_id, additional_info, timing)) 

the last closing bracket here correspond to the first open bracket in line 77, it encloses all data within the brackets, another practice is to put the closing bracket to the rightmost column to enclose all the objects inside